docs/modules/cloud/controller.md

```diff
@@ -0,0 +1,54 @@
+# Edge Controller
+
+
+## Edge Controller Overview
+ Controller(also known as edgecontroller) is the bridge between Kubernetes Api-Server and Edgecore
+
+   
+## Operations Performed By Edge Controller
+ 
+ The following are the functions performed by Edge controller :-
+  - Downstream Controller:Sync add/update/delete event to edgeCore from K8s Api-server
+  - Upstream Controller:sync watch and Update status of resource and events(node, pod and configmap) to K8s Api-server and also subscribe message from edgecore
```

Comment:
Please use consistent naming conventions.
Edgecore or edgeCore or edgecore. (In the previous lines)

Reply:
ok